data/json/recipes/weapon/ranged.json

```diff
@@ -734,6 +734,23 @@
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "stick", 3 ], [ "2x4", 2 ] ], [ [ "bone_sturdy", 3, "LIST" ] ], [ [ "cordage_superior", 2, "LIST" ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "reflexbow",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "fabrication",
+    "skills_required": [ [ "archery", 2 ] ],
+    "difficulty": 4,
+    "time": "8 h",
+    "autolearn": true,
+    "book_learn": [ [ "recipe_bows", 4 ], [ "manual_archery", 5 ] ],
+    "using": [ [ "adhesive", 3 ] ],
+    "proficiencies": [ { "proficiency": "prof_bowyery" }, { "proficiency": "prof_carving" } ],
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "stick", 2 ], [ "2x4", 1 ] ], [ [ "bone_sturdy", 2, "LIST" ] ], [ [ "cordage_superior", 2, "LIST" ] ] ]
```

Comment:
As above, my understanding of the use of bone in composite bow manufacture is that it gets used to reinforce parts of the bow that do not bend such as the handle area and the tips, a bow with just wood and bone is effectively not a composite bow, it requires sinew and/or horn to be a functional composite bow.

Reply:
I'll always defer to the superior research! The recipe I used here is almost an exact duplicate of the heavy composite bow recipe already existing in the game, and I admit I did no additional reading. I did, just now, find <a href=”https://www.britannica.com/technology/bow-and-arrow#ref127134” target=”_blank”><strong>this</strong></a> article on Britannica mentioning the Inuit using " composite bows of wood and bone backed by sinew." It's a pretty tenuous reference, and supports your assertion that at least sinew needs to be there.

I propose three possible solutions:
1- Add horn to the game and update composite bow recipes to require it. Cow horn is mentioned in the Wikipedia article on composite bows as being bad for this job. We've already got sheep, and goats could be added as well. Who doesn't like goats?

2- Remove composite bow recipes entirely until such a time as horn gets added. This seems undesirable, given the long history of hand made composite bows.

3- Require a significant amount of sinew in the recipe and emphasize its use as a fundamental construction material in the description. This is my personal favorite as a short-term solution, which includes the option of adding horn as an alternative later, but I'm definitely not an archery expert.

I also note the woods involved in real composite bows are fairly specific, and we don't seem to have much simulation of different woods in the game right now. Would it be of interest to add something more specialized (I see bamboo mentioned) for the purpose of composite bows?

Reply:
Okay, I came at this thinking I was simply re-using data already accepted into the game for a quick issue resolution but upon further reading I see I'm going to have to get into a pretty complex topic if I want to help move bows toward greater realism. I'm having a hard time finding good info on fully sinew backed composite bows, and it's pretty clearly a niche design which wouldn't be readily known to the average hobbyist.

I'm also not convinced there's a ready supply of good horn from the animals in the area, even if there were added availability of sheep and goat horn since only certain breeds have horns really suitable for 'standard' length composite bows. 

For the time being I think it's best I close this PR and do work in areas more fitting my experience. If it's helpful, it looks like native american bows may be a good source of info on fully sinew backed bows and apparently the Sioux used a bow constructed with horn from mountain sheep and cow, as well as elk antler.